### DISCUSSION TITLE
Welcome to Discussion Space!

### DISCUSSION THREAD

**Discussion by ajshank** [unl-nimbus-lab/Freyja#7] [General]
## Welcome!
  Feel free to use this space to:
  * Ask usage/behaviour questions,
  * Share ideas or features,
  * Engage with other users and developers.

Please report implementation-related errors and problems under **Issues**. This space is more casual and intended for conversation.

**Comment by dunlookme** [comment]
Hello, this is an interesting work. Is it possible to implement the LQR controller to Px4?

**Reply by ajshank** [reply]
Hello @dunlookme, thanks for your interest. Freyja does work with the px4 stack (over `mavros`), by using px4's `OFFBOARD` flight mode. This has been tested only on bench though, and more extensive flight tests are still pending. I'm happy to have more testers for px4, if there's interest.

Freyja is tested mostly on the ArduCopter stack and AscTec vehicles. The convenience handlers/interface for px4, and the recommended config/settings, are not made public here because of this.
